### DISCUSSION TITLE
Kicked out mid reading when wifi dips out

### DISCUSSION THREAD

**Discussion by lukakostic** [mdn/mdn-community#684] [Platform] [answered]
Who thought it was a good idea,   
that in the **middle of reading an already well loaded page**,   
you get kicked off when your wifi craps out for a second (page auto returns after a few seconds, but still) ?

![image](https://github.com/mdn/mdn-community/assets/41348897/41702dd3-adb7-4fba-93f0-6356200cb249)

**Comment by pransh15** [comment, marked as answer]
Interesting! Thanks for sharing this @lukakostic.
I was not able to replicate this on my machine. After jumping onto the same page([link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise)) as yours, I turned my WiFi off to see if it gave me the same error, and it did not. It might have happened due to the ongoing maintenance but I'd like to know if it still gives you this error.

If it still happens, please share more information about your device and browser. I'd like to add an issue and share it with the devs if it does.

**Reply by pransh15** [reply]
Hi @lukakostic 👋 
Let me know if it's still an issue, and we can work on it. If not - I'll close this discussion. 😄

**Comment by bsmth** [comment]
I can't repro this either, so I think we can close for now.
